Sources/ArcGISToolkit/Common/AttachmentsFeatureElementView.swift

```diff
@@ -67,18 +67,33 @@ struct AttachmentsFeatureElementView: View {
     @State private var isExpanded = true
     
     var body: some View {
-        Group {
+        Section {
             switch attachmentModelsState {
             case .notInitialized, .initializing:
                 ProgressView()
                     .padding()
+                    .task {
+                        guard case .notInitialized = attachmentModelsState else { return }
+                        attachmentModelsState = .initializing
+                        
+                        let attachments = (try? await featureElement.featureAttachments) ?? []
+                        
+                        let attachmentModels = attachments
+                            .reversed()
+                            .map {
+                                AttachmentModel(
+                                    attachment: $0,
+                                    displayScale: displayScale,
+                                    thumbnailSize: thumbnailSize
+                                )
+                            }
+                        attachmentModelsState = .initialized(attachmentModels)
+                    }
             case .initialized(let attachmentModels):
                 if isShowingAttachmentsFormElement {
                     // If showing a form element, don't show attachments in
                     // a disclosure group, but also ALWAYS show
-                    // the list of attachments, even if there are none.
-                    attachmentHeader
-                    attachmentBody(attachmentModels: attachmentModels)
+                        attachmentBody(attachmentModels: attachmentModels)
```

Comment:
Please decrease the indentation here.

```suggestion
                    attachmentBody(attachmentModels: attachmentModels)
```